pkg/proxy/util/utils_test.go

```diff
@@ -1183,6 +1183,50 @@ func TestWriteLine(t *testing.T) {
 	}
 }
 
+func TestWriteRuleLine(t *testing.T) {
+	testCases := []struct {
+		name      string
+		chainName string
+		words     []string
+		expected  string
+	}{
+		{
+			name:      "write no line due to no words",
+			chainName: "svcChain",
+			words:     []string{},
+			expected:  "",
+		},
+		{
+			name:      "write no line due to no chain",
+			chainName: "",
```

Comment:
just remove this test case. If someone calls `WriteRuleLine(buf, "", "-j", "FOO")` then we'll output "`-A  -j FOO`" and if that's not what they wanted they they shouldn't have done that!

Reply:
done.